doc-content/enterprise-only/getting-started-kogito/ref-kogito-microservice-deploy-troubleshooting.adoc

```diff
@@ -0,0 +1,111 @@
+[id="ref-kogito-microservice-deploy-troubleshooting_{context}"]
+= {KOGITO} microservice deployment troubleshooting
+
+Use the information in this section to troubleshoot issues that you might encounter when using the operator to deploy {KOGITO} microservices. The following information is updated as new issues and workarounds are discovered.
+
+No builds are running::
+If you do not see any builds running nor any resources created in the relevant namespace, enter the following commands to retrieve running pods and to view the operator log for the pod:
```

Comment:
The operator is cluster wide, usually not available for ordinary user (unless the user has admin rights).
In general, it is better to check the KogitoBuild resource to see its status (in a similar way as done below for KogitoRuntime). Running for example `oc describe KogitoBuilds example` shows you the `example `KogitoBuild` resource. User can then check its status to see possible issues.